### PR TITLE
Schema: skeleton

### PR DESCRIPTION
**Pull request by woodruffw**
This adds a quick draft of the planned top-level and testcase schemas, as Pydantic models.

~~Running the script (i.e. `python schema/models.py`) will produce a corresponding JSON Schema on `stdout`, for the time being.~~

This adds a little self-management tool called `limbo`; `limbo schema` dumps the current top-level schema onto `stdout`.

### CODE REVIEW

**Comment by woodruffw** [comment]
Expanding from #3: given the WIP `limbo` CLI here, what I'm thinking (roughly) is:

```
# (re)generate testcase assets
limbo build-assets -o ./testcases/assets/

# compile all testcases in `./testcases` into `limbo.json`, validating their contents
limbo compile --testcases ./testcases -o ./limbo.json
```

(Where `compile` and `build-assets` are haphazard verbs.)

File: limbo/models.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        ..., description="The expected validation result"
+    )
+
+    expected_peer_names: list[PeerName] = Field(..., description="The expected peer names")
+
+    trusted_certs: list[StrictStr] = Field(
+        ..., description="A list of CA certificates to consider trusted"
+    )
+
+    subject: StrictStr = Field(..., description="The path to the EE or other subject certificate")
```
Comment (alex) [inline limbo/models.py]: Not sure I understand what this field is?

**Review by alex — COMMENTED** [review]
One small comment, but teh schema looks like a good starting point. I'm sure we'll iterate it as we go.

File: limbo/models.py (hunk 1 above)
Comment (woodruffw) [reply limbo/models.py]: Neither did I -- renamed to `peer_certificate`.

**Pull request merged**